tests/functional/graph_selection/test_graph_selection.py

```diff
@@ -0,0 +1,358 @@
+
+
+def test__postgres__specific_model(project):
+    run_dbt(["seed"])
+    results = run_dbt(["run", "--select", "users"])
+    assert len(results) == 1
+
+    table_comp = TableComparison(
+        adapter=project.adapter, unique_schema=project.test_schema, database=project.database
+    )
+    table_comp.assert_tables_equal("seed", "users")
+
+    created_tables = project.get_tables_in_schema()
+    assert "users_rollup" not in created_tables
+    assert "alternative.users" not in created_tables
+    assert "base_users" not in created_tables
+    assert "emails" not in created_tables
+    assert_correct_schemas(project, table_comp)
+
+
+def test__postgres__tags(project, project_root):
+    run_dbt(["seed"])
+    results = run_dbt(["run", "--selector", "bi_selector"])
+    table_comp = TableComparison(
+        adapter=project.adapter, unique_schema=project.test_schema, database=project.database
+    )
+    assert len(results) == 2
+    created_tables = project.get_tables_in_schema()
+    assert not ("alternative.users" in created_tables)
+    assert not ("base_users" in created_tables)
+    assert not ("emails" in created_tables)
+    assert "users" in created_tables
+    assert "users_rollup" in created_tables
+    assert_correct_schemas(project, table_comp)
+    manifest_path = project_root.join("target/manifest.json")
+    assert os.path.exists(manifest_path)
+    with open(manifest_path) as fp:
+        manifest = json.load(fp)
+        assert "selectors" in manifest
+
+
+def test__postgres__tags_and_children(project):
+    run_dbt(["seed"])
+    results = run_dbt(["run", "--select", "tag:base+"])
+    assert len(results) == 5
+    created_models = project.get_tables_in_schema()
+    assert not ("base_users" in created_models)
+    assert not ("emails" in created_models)
+    assert "emails_alt" in created_models
+    assert "users_rollup" in created_models
+    assert "users" in created_models
+    assert "alternative.users" in created_models
+    table_comp = TableComparison(
+        adapter=project.adapter, unique_schema=project.test_schema, database=project.database
+    )
+    assert_correct_schemas(project, table_comp)
+
+
+def test__postgres__tags_and_children_limited(project):
+    run_dbt(["seed"])
+    results = run_dbt(["run", "--select", "tag:base+2"])
+    assert len(results) == 4
+    created_models = project.get_tables_in_schema()
+    assert not ("base_users" in created_models)
+    assert not ("emails" in created_models)
+    assert "emails_alt" in created_models
+    assert "users_rollup" in created_models
+    assert "users" in created_models
+    assert "alternative.users" in created_models
+    assert "users_rollup_dependency" not in created_models
+    table_comp = TableComparison(
+        adapter=project.adapter, unique_schema=project.test_schema, database=project.database
+    )
+    assert_correct_schemas(project, table_comp)
+
+
+def test__postgres__specific_model_and_children(project, create_tables):  # noqa
```

Comment:
Was there a reason for switching most of the tests to use the 'seed' command and the seed file and not the handful which still have 'create_tables'?

Reply:
Was there a reason for needing the 'noqa'?

Reply:
Create_tables is not used in this function so need noqa to get the pre-commit check pass.
I left this here to show that you can do seed in both ways, but if we prefer [this](https://github.com/dbt-labs/dbt-core/blob/8fd8dfcf74657ccc5314640b22cb2589eed8a824/tests/functional/basic/test_simple_reference.py#L140) I can switch all of it over